web-folder/web-app/src/config/themeColor.js

```diff
@@ -0,0 +1,26 @@
+const lightTheme = {
+  primary: {
+    100: "#E5EEEE",
+    200: "#B8DBD9",
+    300: "#586F7C",
+    400: "#2F4550",
+    500: "#1D272C",
+  },
+};
+
+const darkTheme = {
+  primary: {
+    100: "#586F7C",
+    200: "#2F4550",
+    300: "#E5EEEE",
+    400: "#B8DBD9",
+    500: "#E5EEEE",
+  },
+};
+
+const themeColor = {
+  lightTheme,
+  darkTheme,
+};
```

Comment:
From other code maybe change from darkTheme/lightTheme to dark/light so you don't have to + 'theme' each time you called